### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by JJSirius** [ontox-hu/aspis4j#8] [General]
I propose a modification to the Roadmap

Original approach:
1. Create Entities
      - Chemicals
      - Gene
      - Disease
      - Pathways
3. Create Associations/Relationship
      - Chem/Gene, etc 
4. CRUD for entities and associations
5. ETL for entities and associations

Proposed approach
1. Chemical Entity
      - Model
      - CRUD
      - ETL
      - Documentation
2. Gene Entity
      - Model
      - CRUD
      - ETL
      - Documentation
3. Chemical Gene Interaction/Relationship
      - Model
      - CRUD
      - ETL
      - Documentation
4. Graph Visualization 
5. Disease Entity
      - Model
      - CRUD
      - ETL
      - Documentation   
 
And so on...

**Comment by tomlue** [comment]
This looks good to me feel free to update the readme and issues
